sim/core/consumes.go

```diff
@@ -393,7 +393,7 @@ func DragonBreathChiliAura(character *Character) *Aura {
 		ActionID:    ActionID{SpellID: 15851},
 		SpellSchool: SpellSchoolFire,
 		DefenseType: DefenseTypeMagic,
-		ProcMask:    ProcMaskEmpty,
+		ProcMask:    ProcMaskSpellDamage,
```

Comment:
Shouldn't chili be `ProcMaskSpellDamageProc`?

Reply:
changed to ProcMaskSpellDamageProc | ProcMaskSpellProc so it procs both procs that require damage and those that don't.